app/src/main/res/layout/crf_step_layout_photo_capture.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
You don't need a custom UI here since it is the same as the CRFInstructionStepLayout